### DISCUSSION TITLE
SQL Server Vector Store

### DISCUSSION THREAD

**Discussion by dccrain** [microsoft/semantic-kernel#9765] [Q&A] [answered]
I see that SQL Server Connector has a MemoryStore implementation, I'm curious if there is work being done to also release a VectorStore implementation for that connector.

Thanks!

**Comment by sophialagerkranspandey** [comment]
Adding @westey-m

**Comment by westey-m** [comment, marked as answer]
@dccrain, We haven't started work on this yet, but it is on our backlog.  We need to do some prioritization of the remaining gaps we have in our vector store implementations, and we can get back to you once we have our plan.

**Reply by westey-m** [reply]
@dccrain, work for this connector is now in progress: #10623
